pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: matrix
 description: Matrix Dart SDK
-version: 0.27.0
+version: 0.28.0
```

Comment:
Do we have a breaking change here? In my PR I made sure that we only deprecate stuff, but not removing apis

Reply:
yep we merged the huge calls change